Program/program2.py

```diff
@@ -0,0 +1,33 @@
+Python 3.12.4 (tags/v3.12.4:8e8a4ba, Jun  6 2024, 19:30:16) [MSC v.1940 64 bit (AMD64)] on win32
+Type "help", "copyright", "credits" or "license()" for more information.
+>>> import sqlite3 as sq

```

Comment:
why you use import sqlite3 as " sq " ??

Reply:
here we are giving short identification of sqlite3 as sq . when we connect python with  sqlite3 we use this statement conn=sq.connect sq is object. hope you undestand..